app/org/sagebionetworks/bridge/json/ScheduleSerializer.java

```diff
@@ -31,14 +34,23 @@ public void serialize(Schedule schedule, JsonGenerator gen, SerializerProvider p
         gen.writeStartObject();
         writeString(gen, LABEL_PROPERTY, schedule.getLabel());
         writeString(gen, CRON_TRIGGER_PROPERTY, schedule.getCronTrigger());
-        if (schedule.getStartsOn() != null && schedule.getStartsOn() != 0L) {
-            writeString(gen, STARTS_ON_PROPERTY, DateUtils.convertToISODateTime(schedule.getStartsOn()));    
+        if (schedule.getStartsOn() != null) {
+            writeString(gen, STARTS_ON_PROPERTY, DateUtils.getISODateTime(schedule.getStartsOn()));    
         }
-        if (schedule.getEndsOn() != null && schedule.getEndsOn() != 0L) {
-            writeString(gen, ENDS_ON_PROPERTY, DateUtils.convertToISODateTime(schedule.getEndsOn()));    
+        if (schedule.getEndsOn() != null) {
+            writeString(gen, ENDS_ON_PROPERTY, DateUtils.getISODateTime(schedule.getEndsOn()));    
         }
-        if (schedule.getExpires() != null && schedule.getExpires() != 0L) {
-            writeString(gen, EXPIRES_PROPERTY, DateUtils.convertToDuration(schedule.getExpires()));    
+        if (schedule.getExpires() != null) {
+            writeString(gen, EXPIRES_PROPERTY, schedule.getExpires().toString());    
```

Comment:
From the Javadocs, they return HH:mm:ss.SSS and yyyy-MM-ddTHH:mm:ss.SSSZZ respectfully. These are ISO8601 strings, but not all ISO8601 strings are in this format. If this is the format we want, then I'm fine with that. I'd still strongly prefer to explicitly declare a format rather than relying on the default, though, but I'll leave it up to you.